app/src/index.ts

```diff
@@ -8,11 +8,11 @@ export {
   EditorConfiguratorStateProvider,
   useConfiguratorState,
 } from "../configurator";
-export { DatasetBrowser } from "../browser";
-export { LocaleProvider, useLocale } from "../locales/use-locale";
 export {
-  i18n,
   defaultLocale,
+  i18n,
   locales,
   parseLocaleString,
 } from "../locales/locales";
+export { LocaleProvider, useLocale } from "../locales/use-locale";
+export { DatasetBrowser } from "../pages/browse";
```

Comment:
Is this necessary ? I think we should try to limit the number of things we export from a big index module otherwise we increase the risk of importing src and having a circular dependency.